extensions/internet-radio/src/browse-stations.tsx

```diff
@@ -0,0 +1,158 @@
+      const tags = [];
+
+      if (currentStationName == stationName) {
+        // Add "Now Playing" icon
+        tags.push({ icon: Icon.Play });
+      } else if (!isPlaying && lastStationName == stationName) {
+        // Add last played station indicator
+        tags.push({ text: "Last Played" });
+      }
+
+      // Add genre labels
+      for (const genre of stationData.genres) {
+        if (genre == "") continue;
+        let tagColor = Color.SecondaryText;
+        if (genre in colorMap) {
+          tagColor = colorMap[genre];
+        }
+        tags.push({ tag: { value: genre, color: tagColor } });
+      }
+
+      listItems.push(
+        <List.Item
+          key={stationName}
+          icon={getFavicon(stationData.website as string)}
+          title={stationName}
+          accessories={tags}
+          actions={
+            <ActionPanel>
+              <ActionPanel.Section>
+                {!isPlaying || currentStationName != stationName ? (
+                  <Action
+                    title={"Stream with Music"}
+                    icon={Icon.Play}
+                    onAction={() => {
+                      playStation(stationName, stationData.stream as unknown as string).then((stationID) => {
+                        setIsPlaying(true);
+                        setCurrentStationName(stationName);
+                        setCurrentTrackID(stationID);
+                      });
+                      showToast({ title: "Playing Station", message: stationName });
+                    }}
+                  />
+                ) : null}
+
+                {isPlaying && currentStationName == stationName ? (
+                  <Action
+                    title={"Stop Playback"}
+                    icon={Icon.Stop}
+                    onAction={() => {
+                      pausePlayback().then(() => {
+                        deleteTrack(currentTrackID);
+
+                        setIsPlaying(false);
+                        setCurrentStationName("");
+                        setCurrentTrackID("");
+
+                        showToast({ title: "Stopped Station", message: currentStationName });
+                      });
+                    }}
+                  />
+                ) : null}
+              </ActionPanel.Section>
+
+              <ActionPanel.Section>
+                <Action.OpenInBrowser title="Open Stream In Browser" url={stationData.stream as string} />
+                <Action.OpenInBrowser title="Open Website" url={stationData.website as string} />
+                <Action.CopyToClipboard title="Copy Stream URL" content={stationData.stream as string} />
+                <Action.CopyToClipboard title="Copy Website URL" content={stationData.website as string} />
+              </ActionPanel.Section>
+
+              <ActionPanel.Section>
+                <Action
+                  title={"Delete Station"}
+                  style={Action.Style.Destructive}
```

Comment:
Missin an icon

Reply:
Maybe add something like `shortcut={{ modifiers: ["ctrl"], key: "x" }}`

Reply:
It could also be nice if there was a [confirm before deleting](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) 😊